.gitignore

```
@@ -1,2 +1,6 @@
 venv/
-.idea/
\ No newline at end of file
+.idea/
+.vscode/settings.json
+dicom_app/__pycache__/__init__.cpython-37.pyc
```

```suggestion
dicom_app/__pycache__/__init__.cpython-37.pyc
```
Maybe use **.pyc instead if specific folder, same for __pycache__ in whole project

---

Right!